### DISCUSSION TITLE
indoorGML convertable to CityJSON?

### DISCUSSION THREAD

**Discussion by ZhuoranJia** [cityjson/specs#177] [Q&A] [answered]
Hello, can an IndoorGML file be converted to a CityJSON file? Thanks.

**Comment by hugoledoux** [comment, marked as answer]
Depends what you want. If you want only the geometries then yes I guess it would be possible to do so (only FME would work at the moment: read the GML file with it and export to CityJSON).

But if you want semantics then no not possible. The data model of CityJSON is that of CityGML, and the dual network essential to IndoorGML is not one of the classes. You could store each room/floor with the equivalent CityGML v3 classes though: "BuildingStorey" + "BuildingRoom" (see https://www.cityjson.org/specs/1.1.3/#building).

For the dual graph, you could create your own [Extension](https://www.cityjson.org/extensions/).
